src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,219 @@
+                                + "FROM cars c "
+                                + "INNER JOIN manufacturers m "
+                                + "ON c.manufacturer_id = m.id "
+                                + "WHERE c.is_deleted = FALSE";
+        List<Car> listOfCars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                Car car = parseCarFromResultSet(resultSet);
+                listOfCars.add(car);
+            }
+        } catch (SQLException exc) {
+            throw new DataProcessingException(
+                    "Can't get all entries from table Cars", exc);
+        }
+        return listOfCars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "UPDATE cars SET model = ?, manufacturer_id = ?"
+                    + " WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement =
+                        connection.prepareStatement(query)) {
+            preparedStatement.setString(1, car.getModel());
+            preparedStatement.setLong(2, car.getManufacturer().getId());
+            preparedStatement.setLong(3, car.getId());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t update car from table Cars" + car, e);
+        }
+        deleteCarDrivers(car.getId());
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            preparedStatement.execute();
+        } catch (SQLException exc) {
+            throw new DataProcessingException("Couldn't delete car by id = " + id, exc);
+        }
+        return true;
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query =
+                "SELECT c.id id, c.model model, "
+                        + "m.id manufacturer_id, m.name manufacturer, "
+                        + "m.country country "
+                        + "FROM cars c"
+                        + "INNER JOIN cars_drivers c_d "
+                        + "ON cars.id = cars_drivers.car_id "
+                        + "INNER JOIN manufacturers m "
+                        + "ON manufacturers.id = cars.manufacturer_id "
+                        + "WHERE driver_id = ? AND cars.is_deleted = FALSE "
+                        + "AND drivers.is_deleted = FALSE";
```

Comment:
this won't work - you do not join drivers (and there is no need to join them and check this)